pkg/deploy/elbv2/listener_manager.go

```diff
@@ -58,7 +65,10 @@ func (m *defaultListenerManager) Create(ctx context.Context, resLS *elbv2model.L
 	if err != nil {
 		return elbv2model.ListenerStatus{}, err
 	}
-	lsTags := m.trackingProvider.ResourceTags(resLS.Stack(), resLS, resLS.Spec.Tags)
+	lsTags := map[string]string{}
```

Comment:
maybe `var lsTags  map[string]string` instead, which avoids the unnecessary map allocation

Reply:
done.